resources/nl-core/nl-core-NameInformation.xml

```diff
@@ -35,7 +33,100 @@
   <differential>
     <element id="HumanName">
       <path value="HumanName" />
+      <comment value="This `.name` element accomodates the official parts of a Dutch name according to common international usage and optionally to the [zib NameInformation (v1.1, 2020)](https://zibs.nl/wiki/NameInformation-v1.1(2020EN)). An official Dutch name is represented in FHIR as an ordinary international name, optionally augmented using extensions to specify how the last name is built up according to the Dutch rules if conformance to the zib is required. See the guidance on `.family` and on `.extension:nameUsage` for more information.&#xD;&#xA;&#xD;&#xA;Systems that need to work in a Dutch context SHOULD support the extensions as specified in `.given`, `.family` and slice `nameUsage`. In addition, systems SHOULD use the core elements according to the FHIR specifications to provide compatibility outside Dutch contexts. This means that:&#xD;&#xA;* If parts of the name are specified using one or more of the structured elements, the `.text` element SHOULD be populated with a rendered version of the full name.&#xD;&#xA;* If parts of the family name are specified using the extensions in `.family`, the `.family` element SHOULD be populated with the full family name.&#xD;&#xA;&#xD;&#xA;**Note 1**: The zib cannot be represented straightforward in FHIR. Especially note the guidance on `.given` on how to map the FirstNames and Initials concepts, and on `.prefix`/`.suffix` on how to map the Titles concept.&#xD;&#xA;&#xD;&#xA;**Note 2**: This element should only contain a person's _official_ names. The GivenName concept is represented in another `.name` element with `.name.use` = _usual_.&#xD;&#xA;&#xD;&#xA;**Note 3**: The examples illustrate how the zib is mapped to FHIR." />
       <alias value="nl-core-NameInformation" />
+      <constraint>
+        <key value="nl-core-NameInformation-01" />
+        <severity value="warning" />
+        <human value="If parts of the name are specified using one or more of the structured elements, the `.text` element SHOULD be populated with a rendered version of the full name. This ensures compatibility with common FHIR usage." />
+        <expression value="(family.exists() or given.extension.where(url = 'http://hl7.org/fhir/StructureDefinition/iso21090-EN-qualifier').where(value != 'IN').exists()) implies text.hasValue()" />
+      </constraint>
+    </element>
+    <element id="HumanName.text">
+      <path value="HumanName.text" />
+      <comment value="A representation of the full name, including given names and the familiy name. Although this element represents the official names, it is considered desirable to include unofficial names as well if commonly used, for example added in brackets.&#xD;&#xA;&#xD;&#xA;When implementing zib NameInformation, this rendition should include GivenName or Initials, the GivenName and the family name constructed according to the NameUsage concept." />

```

Comment:
To me, the word 'rendition' is not very clear. Can we change that word (format or representation?) or I think we can even do without that word? 

But this is definitely just a cosmetic comment.

Reply:
Should we mention here that the GivenName is provided in a sibling element, like we are stating in the mappings below?

Reply:
Removed the whole mention of GivenName from this guidance because of ZIBFHIR-54.

Also, removed the word "rendition".